src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -156,7 +156,7 @@ public static class TreeExtensions
         }
 
         /// <summary>
-        /// Ranks a series of inputs based on their relevance, training a decision tree ranking model with the <see cref="FastTreeRankingTrainer"/> and advanced options.
+        /// Create a <see cref="FastTreeRankingTrainer"/> with advanced options, which ranks a series of inputs based on their relevance and training a decision tree ranking model.
```

Comment:
Same comment as above. Suggest: which ranks a series of inputs based on their relevance, using a decision tree ranking model